packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/CSharpGen.cs

```diff
@@ -45,6 +45,13 @@ public async Task ExecuteAsync()
                 generateFilesTasks.Add(workspace.AddGeneratedFile(Path.Combine("src", "Generated", $"{client.Name}.cs"), writer.ToString()));
             }
 
+            foreach (var helper in output.Helpers)
+            {
+                CodeWriter writer = new();
+                CodeModelPlugin.Instance.GetExpressionTypeProviderWriter(writer, helper).Write();
```

Comment:
any thoughts on simplifying this method name? `GetExpressionTypeProviderWriter` is a little bit too verbose. Just `GetWriter` should work fine. Not to mention we actually no longer have a thing called `ExpressionTypeProvider` now.

Reply:
Opened a PR here: https://github.com/microsoft/typespec/pull/3334